client.js

```diff
@@ -15,31 +15,35 @@ let client;
 
 async function lookup({ _, ...rest }) {
   if (!rest.id) {
-    console.log("fetch required an ID");
+    console.log("lookup requires an ID");
     process.exit();
   }
-  console.log("Beginning client-side lookup: ", rest.id);
-  const lookedUpUser = await client.lookup({ id: rest.id });
-  console.log(lookedUpUser);
-  if (!lookedUpUser) {
-    console.log("Finished client-side lookup, found nothing");
-  } else {
-    console.log("Finished client-side lookup: ", lookedUpUser);
-  }
+
+  await client.lookup({ id: rest.id }, (err, user) => {
+    if (err) {
+      console.log("User not found");
+      console.log(err);
+    } else {
+      console.log("User found: ", user);
+    }
+  });
 }
 
-function remove({ _, ...rest }) {
+async function remove({ _, ...rest }) {
   if (!rest.id) {
-    console.log("remove required an ID");
+    console.log("remove requires an ID");
     process.exit();
   }
   console.log("Beginning client-side remove: ", rest.id);
-  try {
-    client.remove({ id: rest.id });
-    console.log("Remove successful, finishing");
-  } catch (err) {
-    conosle.log("Remove failed, err: ", err);
-  }
+
+  await client.remove({ id: rest.id }, (err, _) => {
+    if (err) {
+      console.log("User not deleted");
```

Comment:
I only hit this condition if trying to delete a user that I've never inserted. I get repeated success when I delete a user that I have inserted. I would expect RC5 when I delete users I've already deleted (and which return RC5 on fetch)

Reply:
Good catch, I skipped this scenario in my testing, I was returning always null as error :) probably because I know our users are so smart they never delete things twice!